src/ContaoCommunityAlliance/DcGeneral/Contao/Dca/Populator/BackendViewPopulator.php

```diff
@@ -159,6 +160,11 @@ protected function populatePanel(EnvironmentInterface $environment)
                 }
             }
         }
+
+        $baseConfig    = $environment->getBaseConfigRegistry()->getBaseConfig();
+        $listingConfig = $viewDefinition->getListingConfig();
+
+        ViewHelpers::initializeSorting($panel, $baseConfig, $listingConfig);
```

Comment:
I'm not sure this is the right place to add this. The populators are meant to build the environment and not to perform functional tasks.
By this change, it is impossible to manipulate the sorting in another listener as the sorting already has been set.